tmt/steps/prepare/artifact/providers/koji.py

```diff
@@ -219,6 +220,18 @@ def _download_artifact(
         except Exception as error:
             raise DownloadError(f"Failed to download '{artifact}'.") from error
 
+    def contribute_to_shared_repo(
+        self,
+        guest: Guest,
+        download_path: tmt.utils.Path,
+        shared_repo_dir: tmt.utils.Path,
+        exclude_patterns: Optional[list[tmt.utils.Pattern[str]]] = None,
+    ) -> None:
+        guest.execute(
+            ShellScript(f"cp -r {quote(str(download_path))}/. {quote(str(shared_repo_dir))}")
```

Comment:
```suggestion
            ShellScript(f"cp {quote(str(download_path))}/*.rpm {quote(str(shared_repo_dir))}")
```
We can be more eplicit for these, just in case we would want to have some other logs or such in that staging directory.